zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -584,26 +611,19 @@ mod tests {
             r#"CREATE INDEX orchard_received_notes_account ON orchard_received_notes (
                 account_id ASC
             )"#,
-            r#"CREATE INDEX orchard_received_notes_spent ON orchard_received_notes (
```

Comment:
Should these index removals be replaced with index creations for the foreign keys of the new junction tables?